hilti/runtime/include/types/integer.h

```diff
@@ -99,7 +99,9 @@ inline Bytes pack(integer::safe<T> i, ByteOrder fmt) {
     if ( fmt == ByteOrder::Host )
         return pack<T>(i, systemByteOrder());
 
-    uint8_t raw[sizeof(T)];
+    std::string str;
+    str.resize(sizeof(T));
```

Comment:
You could allocate a string of the right size on declaration to avoid a potential realloc, and since we want `Bytes` which are just a small wrapper around `std::string` you could directly construct the `Bytes` we return (should remove the `std::move` on `return`),

```suggestion
    Bytes str(sizeof(T), '\0');
```

Reply:
Nice, folded that in.